templates/CRM/Custom/Form/Field.tpl

```diff
@@ -229,7 +229,7 @@
 
     function showSearchRange(dataType) {
       if (_.includes(['Date', 'Int', 'Float', 'Money'], dataType)) {
-        $("#searchByRange", $form).toggle($('#is_searchable', $form).is(':checked'));
+        $("#searchByRange", $form).show();
```

Comment:
I'm not sure exactly what 'search by range' does, but it used to show only if the 'searchable' box was ticked.  Is this change intentional?

Reply:
@artfulrobot - got it, thanks.

Reply:
This field doesn't seem to keep its value on save. I don't think it's from this javascript change - maybe it broke somewhere else?

Reply:
Oh I see. It's sort of related to this change, because this field is tied to "is searchable" in postprocess: https://github.com/civicrm/civicrm-core/blob/a28c91a0694a23bb22591198d451de91d55d4820/CRM/Custom/Form/Field.php#L844-L845

So if you don't check is_searchable, but check this now always-viewable range field, then it ignores you.

Reply:
@demeritcowboy ooh thanks - fixed